sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -526,13 +526,12 @@ def test_dataframe_cov_corr(self):
     df = pd.DataFrame(np.random.randn(20, 3), columns=['a', 'b', 'c'])
     df.loc[df.index[:5], 'a'] = np.nan
     df.loc[df.index[5:10], 'b'] = np.nan
-    self._run_test(lambda df: df.corr().round(8), df)
-    self._run_test(lambda df: df.cov().round(8), df)
-    self._run_test(lambda df: df.corr(min_periods=12).round(8), df)
-    self._run_test(lambda df: df.cov(min_periods=12).round(8), df)
-    self._run_test(lambda df: df.corrwith(df.a).round(8), df)
-    self._run_test(
-        lambda df: df[['a', 'b']].corrwith(df[['b', 'c']]).round(8), df)
+    self._run_test(lambda df: df.corr(), df)
+    self._run_test(lambda df: df.cov(), df)
+    self._run_test(lambda df: df.corr(min_periods=12), df)
+    self._run_test(lambda df: df.cov(min_periods=12), df)
+    self._run_test(lambda df: df.corrwith(df.a), df)
+    self._run_test(lambda df: df[['a', 'b']].corrwith(df[['b', 'c']]), df)
```

Comment:
Because there is logic being modified for the Pearson correlation, can you please add some tests ensuring commutativity between the order of the columns?

Reply:
I'm not sure what you mean by that - do you mean I should add some more tests to verify this works correctly when columns are reordered? Something like:
```
df[['c', 'a', 'b']].corr()
```

Reply:
Right so something like:
pd.testing.assert_frame_equal(pd.DataFrame({'a': [1, 2, 3], 'b': [1, 2, 4]}).corr(), pd.DataFrame({'a': [1, 2, 4], 'b': [1, 2, 3]}).corr()).

Reply:
I don't see the value in that, isn't that just verifying pandas, and not our implementation?

Reply:
That would be true if there was only logic that wrapped the DataFrame.corr method, but there is manual logic here that unwraps the columns into every 2-combination and correlates those. It's a non-blocking issue so up to you.